tests/python_client/testcases/test_collection.py

```diff
@@ -4496,3 +4496,125 @@ def test_create_collection_multiple_vectors_invalid_dim(self, get_invalid_dim):
         error = {ct.err_code: 65535, ct.err_msg: "Invalid dim"}
         self.collection_wrap.init_collection(c_name, schema=schema, check_task=CheckTasks.err_res, check_items=error)
 
+
+class TestCollectionMmap(TestcaseBase):
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_describe_collection_mmap(self):
+        """
+        target: enable or disable mmap in the collection
+        method: enable or disable mmap in the collection
+        expected: description information contains mmap
+        """
+        self._connect()
+        c_name = cf.gen_unique_str(prefix)
+        collection_w, _ = self.collection_wrap.init_collection(c_name, schema=default_schema)
+        collection_w.set_properties({'mmap.enabled': True})
+        pro = collection_w.describe().get("properties")
+        assert "mmap.enabled" in pro.keys()
+        assert pro["mmap.enabled"] == 'True'
+        collection_w.set_properties({'mmap.enabled': False})
+        pro = collection_w.describe().get("properties")
+        assert pro["mmap.enabled"] == 'False'
+        collection_w.set_properties({'mmap.enabled': True})
+        pro = collection_w.describe().get("properties")
+        assert pro["mmap.enabled"] == 'True'
+
+    @pytest.mark.tags(CaseLabel.L1)
+    @pytest.mark.xfail(reason="issue: #30800")
+    def test_load_mmap_collection(self):
+        """
+        target: after loading, enable mmap for the collection
+        method: 1. data preparation and create index
+        2. load collection
+        3. enable mmap on collection
+        expected: raise exception
+        """
+        self._connect()
+        c_name = cf.gen_unique_str(prefix)
+        collection_w, _ = self.collection_wrap.init_collection(c_name, schema=default_schema)
+        collection_w.insert(cf.gen_default_list_data())
+        collection_w.create_index(ct.default_float_vec_field_name, index_params=ct.default_flat_index,
+                                  index_name=ct.default_index_name)
+        collection_w.load()
```

Comment:
Could add a case of setting mmap before load and after insert or create index.

Reply:
> Could add a case of setting mmap before load and after insert or create index.

ok